tools/mo/openvino/tools/mo/moc_frontend/extractor.py

```diff
@@ -6,100 +6,105 @@
 from openvino.tools.mo.front.extractor import raise_no_node, raise_node_name_collision
 from openvino.tools.mo.utils.error import Error
 
-from openvino.frontend import InputModel  # pylint: disable=no-name-in-module,import-error
+from openvino.frontend import (
```

Comment:
unnecessary change, provides no logical difference, premature optimization

Reply:
It is result of using auto formatter `autopep8` on IDE

Reply:
Restored to the previous version(before using auto formatter)